vms/avm/wallet_service.go

```diff
@@ -44,13 +46,11 @@ func (w *WalletService) decided(txID ids.ID) {
 			return
 		}
 
-		err := w.vm.mempool.Add(tx)
-		if err == nil {
+		err := w.vm.network.IssueVerifiedTx(context.TODO(), tx)
+		if err == nil || errors.Is(err, mempool.ErrDuplicateTx) {
 			w.vm.ctx.Log.Info("issued tx to mempool over wallet API",
```

Comment:
nit: not sure we should emit this log for a duplicate?

Reply:
that was the prior behavior... but I think it's fine to only log if it is actually a new tx